envparse.py

```diff
@@ -183,6 +199,8 @@ def read_envfile(path=None, **overrides):
             with open(path, 'r') as f:
                 content = f.read()
         except getattr(__builtins__, 'FileNotFoundError', IOError):
+            # Append the path we tried (and failed) to find to a tracking table
+            traversed_paths = Env.track_path(path)
```

Comment:
Where it's used inside a static method, so there's no `cls` binding param AFAIK.